### PR TITLE
feat(DTFS2-5659): extend component tests coverage TFM UI

### PR DESCRIPTION
**Pull request by vamshidharukef**
## Introduction ✏️ 
Improve coverage for component tests for TFM UI

## Resolution ✔️ 
Create more component tests for TFM UI

### CODE REVIEW

File: trade-finance-manager-ui/component-tests/case/deal/default-view-deal.component-test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// import { page, pageRenderer } from '../../templates/deals/deals.component-test';
```
Comment (lmarrai-sw) [inline trade-finance-manager-ui/component-tests/case/deal/default-view-deal.component-test.js:1]:
```suggestion
```
We can remove this line

**Review by lmarrai-sw — APPROVED** [review]
Thanks for this @vamshidharukef

**Review by lmarrai-sw — DISMISSED** [review]

File: trade-finance-manager-ui/component-tests/case/deal/default-view-deal.component-test.js (hunk 1 above)
Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/case/deal/default-view-deal.component-test.js:1]: I will remove in next PR.
Thanks

**Comment by vamshidharukef** [comment]
https://github.com/UK-Export-Finance/dtfs2/pull/2324 has all the changes similar to PR https://github.com/UK-Export-Finance/dtfs2/pull/2294 so closing this PR.
Thanks

**Pull request closed without merging**